src/carts/src/carts-service/handlers.go

```diff
@@ -8,11 +8,26 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"log"
 	"net/http"
+	"os"
+	"path"
 
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/eventbridge"
 	"github.com/gorilla/mux"
 )
 
+var EventBus = os.Getenv("EVENT_BUS")
```

Comment:
where do you set EVENT_BUS ?

Reply:
In the CloudFormation. This was done in #1 .